app/frontend/src/components/admin/ServiceClients.vue

```diff
@@ -0,0 +1,231 @@
+        <template v-slot:expanded-item="{ headers, item }">
+          <td :class="[responsiveCell]"></td>
+          <td :colspan="headers.length - 1" :class="[responsiveCell]">
+            <div class="kc-expanded">
+              <!-- app details -->
+              <strong>Name: </strong>
+              <span>{{ item.name }}</span>
+            </div>
+
+            <div class="kc-expanded">
+              <!-- app details -->
+              <strong>Created: </strong>
+              <span>{{ formatDate(item.acronymDetails.createdAt) }}</span>
+            </div>
+
+            <div class="kc-expanded">
+              <!-- users -->
+              <strong>Users: </strong>
+              <ul>
+                <li
+                  v-for="(user, index) in item.users"
+                  :key="index"
+                >{{ user.user.firstName + ' ' + user.user.lastName + ((item.users.length > (index + 1)) ? ',' : '') }}</li>
+              </ul>
+            </div>
+          </td>
+        </template>
+      </v-data-table>
+    </v-container>
+  </v-card>
+</template>
+
+<script>
+import keycloakService from '@/services/keycloakService';
+
+export default {
+  name: 'Clients',
+  computed: {
+    responsiveCell () {
+      return (this.$vuetify.breakpoint.name == 'xs') ? 'v-data-table__mobile-table-row' : '';
+    }
+  },
+  data() {
+    return {
+      // vuetify data table
+      search: '',
+      headers: [
+        { text: '', value: 'data-table-expand' },
+        { text: 'Application', align: 'start', value: 'acronym' },
+        { text: 'DEV', value: 'dev' },
+        { text: 'TEST', value: 'test' },
+        { text: 'PROD', value: 'prod' }
+      ],
+      serviceClients: [],
+      loading: true,
+      showAlert: false,
+      alertType: null,
+      alertMessage: '',
+      expanded: []
+    };
+  },
+  methods: {
+
+    formatEnvDate(env) {
+      return (env && env.created) ?  this.formatDate(env.created): '-';
+    },
+    formatDate(date){
+      return new Date(date).toLocaleString();
+    },
+    // get table data from frontend service layer
+    getData() {
+      keycloakService
+        .getServiceClients()
+        .then(response => {
+          const data = response.data;
+          const clients = Object.keys(data).map(k => {
+            let client = data[k];
+            return {
+              acronym: client.acronym,
+              name: client.name,
+              acronymDetails: client.acronymDetails,
+              dev: client.environments.DEV,
+              test: client.environments.TEST,
+              prod: client.environments.PROD,
+              environments: client.environments,
+              users: client.users
+            };
+          });
+
+          if (clients.length == 0) {
+            this.showTableAlert('info', 'No Service Clients found');
+          }
+          this.serviceClients = clients;
+        })
+        .catch(() => {
+          this.showTableAlert('error', 'No response from server');
```

Comment:
Please add a //TODO note regarding global error notification behavior as this will be relevant if/when we tackle that refactor.